enterprise/backend/test/metabase_enterprise/search/scoring_test.clj

```diff
@@ -149,3 +150,33 @@
     (testing "includes both if has both features"
       (mt/with-premium-features #{:official-collections :content-verification}
         (is (set/subset? #{"official collection score" "verified"} (score-result-names)))))))
+
+(deftest fulltext-official-collection-test
+  (fulltext.scoring-test/with-index-contents
+    [{:model "collection" :id 1 :name "collection normal" :official_collection false}
+     {:model "collection" :id 2 :name "collection official" :official_collection true}]
+    (testing "official collections has higher rank"
+      (mt/with-premium-features #{:official-collections}
+        (is (= [["collection" 2 "collection official"]
+                ["collection" 1 "collection normal"]]
+               (fulltext.scoring-test/search :official-collection "collection")))))
+    (testing "only if feature is enabled"
+      (mt/with-premium-features #{}
+        (is (= [["collection" 1 "collection normal"]
+                ["collection" 2 "collection official"]]
```

Comment:
These are more cases where we're relying on search being stable. It probably won't be with other backends.

I guess we can cross that bridge when we get to it, I don't have a good solution besides trading off against another weaker weight, which creates ranker coupling in these "unit" tests.

Why not keep the same approach as for the non-EE tests, and just assert that the order is different? Then in that helper we can mention this future issue, and save our future selves some head scratching.